docs/src/components/demo/Demo.tsx

```diff
@@ -66,14 +70,13 @@ export function Demo({ className, defaultOpen = false, title, ...props }: DemoPr
 
             <Collapsible.Panel
               hidden={false}
+              aria-hidden={!open}
```

Comment:
Shouldn't this be handled by Collapsible itself?

Reply:
Collapsible does the `hidden` attribute out of the box which hides the panel altogether and handles the accessibility.

In this case I needed it to be visibly rendered on the page, but hidden from the accessibility tree when it's not expanded, so I had to add `aria-hidden`

Reply:
gotcha!